### ISSUE TITLE
[EditText] css bug

### ISSUE THREAD

**Issue by DevYuns** [dooboolab/dooboo-ui#254]
**Describe the bug**
[EditText] seems to have css compile error. In project code, `renderContainer` has `paddingVertical: 8` when decoration is `boxed`.

<img width="591" alt="Screen Shot 2023-03-07 at 12 11 58 PM" src="https://user-images.githubusercontent.com/58724686/223311217-2f546ad4-87c0-4d66-951d-fab43499e0d0.png">

But In my `node_modules`, It compiles to `paddingTop: 8`.

<img width="1249" alt="Screen Shot 2023-03-07 at 12 12 09 PM" src="https://user-images.githubusercontent.com/58724686/223311371-bc634a1a-1b95-432c-b543-989a6d0ba29a.png">

I'm currently using `"dooboo-ui": "^0.2.0-rc.6"`.

**Additional context**

**Comment by DevYuns** [comment]
Anyone who try to remove unwanted `paddingTop`,  add `container` style like below.

```ts
<EditText
  decoration="boxed"
  styles={{container: {paddingTop: 0}}}
/>
```

**Comment by hyochan** [comment]
Looks like the original code is `paddingTop: 8`. Closing due to invalid.

**Comment by hyochan** [comment]
However, this looks like `paddingTop` should be `8` only when `label` is provided. Will fix this.

**Comment by hyochan** [comment]
Fixed in 071e299cacffb5fed379f615410ad854eea9bd21
